tensorflow_datasets/image/tiered_imagenet.py

```diff
@@ -0,0 +1,136 @@
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from absl import logging
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+import io
+import os
+import numpy as np
+import pickle as pkl
+from tqdm import tqdm
+import cv2

```

Comment:
You should use lazy_imports to extra dependencies.[link](https://www.tensorflow.org/datasets/add_dataset#extra_dependencies)
You should call cv2 like `tfds.core.lazy_imports.cv2 `